### ISSUE TITLE
When a user picks a single ZIM archive with File System Access API, they cannot then relaunch the same archive by clicking on it in file selector

### ISSUE THREAD

**Issue by Jaifroid** [kiwix/kiwix-js#1382]
This is probably because we use an onchange event rather than an onclick event. It would be very convenient not to have to re-pick the archive. NB this bug does not affect archives shown in the folder view, i.e. when a user has picked a directory of ZIM archives.
